src/uwtools/tests/config/formats/test_fieldtable.py

```diff
@@ -1,39 +1,48 @@
-# pylint: disable=missing-function-docstring
+# pylint: disable=missing-function-docstring,redefined-outer-name
 """
 Tests for uwtools.config.formats.fieldtable module.
 """
 
+from pytest import fixture, mark
+
 from uwtools.config.formats.fieldtable import FieldTableConfig
 from uwtools.tests.support import fixture_path
 from uwtools.utils.file import FORMAT
 
 # Tests
 
 
-def test_get_format():
+@fixture(scope="module")
+def config():
+    return fixture_path("FV3_GFS_v16.yaml")
+
+
+@fixture(scope="module")
+def ref():
+    with open(fixture_path("field_table.FV3_GFS_v16"), "r", encoding="utf-8") as f:
+        return f.read().strip()
+
+
+def test_fieldtable_get_format():
     assert FieldTableConfig.get_format() == FORMAT.fieldtable
 
 
-def test_get_depth_threshold():
+def test_fieldtable_get_depth_threshold():
     assert FieldTableConfig.get_depth_threshold() is None
 
 
-def test_instantiation_depth():
+def test_fieldtable_instantiation_depth():
     # Any depth is fine.
     assert FieldTableConfig(config={1: {2: {3: 4}}})
 
 
-def test_simple(tmp_path):
-    """
-    Test reading a YAML config object and generating a field table file.
-    """
-    cfgfile = fixture_path("FV3_GFS_v16.yaml")
+@mark.parametrize("func", [repr, str])
+def test_fieldtable_repr_str(config, func, ref):
+    assert func(FieldTableConfig(config=config)).strip() == ref
```

Comment:
New `repr()` and `str()` unit tests, which are implemented similarly in the test modules for other `Config` subclasses, below.